### PR TITLE
Darksight Fix + 2 New Traits

### PR DESCRIPTION
**Pull request by Arokha**
A demonstration of all 3: Glowing eyes trait, darksight allows eyes to adjust, and glowing body trait.

In particular, your eyes adjust based on how bright the turf is, and how much darksight you have. If you're going through maint and you stand next to someone with a flashlight, your eyes will unadjust to the darkness and you'll have to let them adjust again when you walk away.

Many thanks to @Leshana for figuring out the particular method of CHEATING BLENDING that can be used for darksight fixing!

### CODE REVIEW

**Comment by Arokha** [comment]
Made Pol PR so removed label.

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 1]
```diff
@@ -527,9 +529,50 @@ var/global/list/damage_icon_parts = list()
...
+
+	//Get the head, we'll need it later.
+	var/obj/item/organ/external/head/head_organ = get_organ(BP_HEAD)
+	if(!head_organ || head_organ.is_stump() )
+		if(update_icons)   update_icons_layers()
+		return
+
+	//The eyes store the color themselves, funny enough.
+	var/obj/item/organ/internal/eyes/eyes = internal_organs_by_name[O_EYES]
+	if(!head_organ.eye_icon)
```
Comment (Leshana) [inline code/modules/mob/living/carbon/human/update_icons.dm]: Looks like this if statement can be eliminated by adding `|| !head_organ.eye_icon` to line 553.

File: code/modules/mob/mob_planes.dm [hunk 2]
```diff
@@ -129,12 +130,17 @@
 //Lighting is weird and has matrix shenanigans. Think of this as turning on/off darkness.
 /obj/screen/plane_master/fullbright
 	plane = PLANE_LIGHTING
-	layer = LIGHTING_LAYER
+	layer = LIGHTING_LAYER+1
 	color = null //To break lighting when visible (this is sorta backwards)
-	alpha = 255 //Starts full opaque
-	invisibility = 101 //But invisible
+	alpha = 0 //Starts full opaque
```
Comment (Leshana) [inline code/modules/mob/mob_planes.dm]: This comment is now a lie

**Comment by LittleBigKid2000** [comment]
Cool, now my Vulp synth's glowing eyes can be represented by his sprite.

File: code/modules/mob/living/carbon/human/update_icons.dm (hunk 1 above)
Comment (Arokha) [reply code/modules/mob/living/carbon/human/update_icons.dm]: I'll change both of...

File: code/modules/mob/mob_planes.dm (hunk 2 above)
Comment (Arokha) [reply code/modules/mob/mob_planes.dm]: these on Polaris.

**Pull request merged**